<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addsub</class>
 <widget class="QDockWidget" name="addsub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Baskerville Old Face</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>450</width>
      <height>721</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="manage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>451</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Baskerville Old Face</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manage Matro</string>
      </property>
     </widget>
     <widget class="QLabel" name="manimg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #D9D9D9;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="metroimg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>630</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>650</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BAXAUTRIAL</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Egypt Metro</string>
     </property>
    </widget>
    <widget class="QLabel" name="adminimg">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Baskerville Old Face</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Baskerville Old Face</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Home Page</string>
     </property>
    </widget>
    <widget class="QLabel" name="imghome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>49</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #BDBDBD;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
border-radius: 12;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Baskerville Old Face</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Tickets</string>
     </property>
    </widget>
    <widget class="QLabel" name="ticimg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>49</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>450</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Baskerville Old Face</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
</string>
     </property>
     <property name="text">
      <string>Subscription</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>530</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Baskerville Old Face</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Station</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="logout">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>630</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BAXAUTRIAL</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFC200;
border-radius: 12;
color: black;</string>
     </property>
     <property name="text">
      <string>Log out</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #BDBDBD;
border-radius: 12;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>220</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville Old Face</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="rides">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>300</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville Old Face</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville Old Face</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville Old Face</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NO. of Rides:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville Old Face</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Subscription</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>550</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BAXAUTRIAL</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFC200;
border-radius: 12;
color: black;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="month">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville Old Face</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>390</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville Old Face</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Duration:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>380</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville Old Face</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Month(s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDBDBD;
border-radius: 5;
border-image: url(:/images/img/left-arrow.png) 0 0 0 0 stretch stretch;
width: 50px;
height: 50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
